ui/src/components/chat/compose.rs

```diff
@@ -26,11 +26,24 @@ struct ComposeData {
     platform: Platform
 }
 
+impl PartialEq for ComposeData {
+    fn eq(&self, _other: &Self) -> bool {
+        false
```

Comment:
does this just mean an == returns false no matter what?

looking at this: https://doc.rust-lang.org/std/cmp/trait.PartialEq.html it seems we can compare objects and return true if specific keys are equal, but i am also don't understand this yet

Reply:
good question. this is a hack. to put something in a struct which implements `Props` via `#[derive(Props)]`, it needs to be comparable. Dioxus will re-render an Element if the props change. 

`ComposeData` can't automatically implement `PartialEq` but i know that any time the parent Element is created, I need to re-render the children. I opted to implement PartialEq but make it always false to communicate that the element will always re-render.